Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -46,6 +47,9 @@ class FontDescription {
     bool operator==(const FontDescription&) const = default;
 
     float computedSize() const { return m_computedSize; }
+    float usedZoomFactor() const { return m_usedZoomFactor; }
+    float computedSizeForRangeZoomOption(CSS::RangeZoomOptions option) const { return (enableEvaluationTimeZoom() && option == CSS::RangeZoomOptions::Unzoomed) ? unzoomedComputedSize() : computedSize(); }
+    float unzoomedComputedSize() const { return m_computedSize / m_usedZoomFactor; }
```

Comment:
mega nit: move this above `computedSizeForRangeZoomOption` so when reading you can see all the dependencies